network-api/networkapi/urls.py

```diff
@@ -41,14 +41,12 @@
     url(r'^wagtail/', include(wagtail_urls))
     if settings.ENABLE_WAGTAIL else None,
 
+    url(r'^fellowship/(?P<path>.*)', RedirectView.as_view(
+        url='/fellowships/%(path)s',
+        query_string=True
+    )),
```

Comment:
not sure if I need to add an extra line in between these 2 lines?

Reply:
nope, not unless pep8 says we need it

Reply:
weird, thought i deleted it 👀